README.md

```diff
@@ -100,7 +100,7 @@ client.login('token');
 ## Links
 
 - [Website](https://discord.js.org/) ([source](https://github.com/discordjs/website))
-- [Documentation](https://discord.js.org/#/docs)
+- [Documentation](https://discord.js.org/docs)
```

Comment:
Shouldn't this be <https://old.discordjs.dev/#/docs>?

Reply:
Okey